src/panels/history/ha-panel-history.ts

```diff
@@ -147,6 +154,11 @@ class HaPanelHistory extends LitElement {
     }
   }
 
+  private _refreshHistory() {
+    this._stateHistory = [];
```

Comment:
is it needed to clear it first?

Reply:
TBH did not give it that much thought, just felt logical to me and the logbook panel did something similar although there it has some dedicated caching I think.

Will remove it, since it works without here.